From b55cd7666130cae7f301ae995e47a5ce3a3aeacf Mon Sep 17 00:00:00 2001
From: Aurimas Liutikas <aurimas@google.com>
Date: Tue, 9 Oct 2018 23:51:48 +0000
Subject: [PATCH 2/3] Fix casting warnings in ContactSaveService

packages/apps/Contacts/src/com/android/contacts/ContactSaveService.java:1227: warning: non-varargs call of varargs method with inexact argument type for last parameter;
                    R.string.contacts_deleted_many_named_toast, names);
                                                                ^
  cast to Object for a varargs call
  cast to Object[] for a non-varargs call and to suppress this warning
1 warning

Test: make -j Contacts (no more warnings)
Change-Id: I244f376b2a8728598e66757742234fbf1a3557df
---
 src/com/android/contacts/ContactSaveService.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/com/android/contacts/ContactSaveService.java b/src/com/android/contacts/ContactSaveService.java
index 4e2d17896..3f645aed0 100755
--- a/src/com/android/contacts/ContactSaveService.java
+++ b/src/com/android/contacts/ContactSaveService.java
@@ -1218,13 +1218,13 @@ public class ContactSaveService extends IntentService {
                     R.plurals.contacts_deleted_toast, contactIds.length);
         } else if (names.length == 1) {
             deleteToastMessage = getResources().getString(
-                    R.string.contacts_deleted_one_named_toast, names);
+                    R.string.contacts_deleted_one_named_toast, (Object[]) names);
         } else if (names.length == 2) {
             deleteToastMessage = getResources().getString(
-                    R.string.contacts_deleted_two_named_toast, names);
+                    R.string.contacts_deleted_two_named_toast, (Object[]) names);
         } else {
             deleteToastMessage = getResources().getString(
-                    R.string.contacts_deleted_many_named_toast, names);
+                    R.string.contacts_deleted_many_named_toast, (Object[]) names);
         }
 
         mMainHandler.post(new Runnable() {
-- 
2.17.1

